### DISCUSSION TITLE
`react-international-phone` v2

### DISCUSSION THREAD

**Discussion by ybrusentsov** [goveo/react-international-phone#39] [Announcements]
I am excited to announce the v2 release of `react-international-phone`! 🎉
This new version brings several improvements and changes that I believe will enhance your experience with the library.

❗ v2 contains several breaking changes so you need to follow the [migration guide](https://react-international-phone-docs.vercel.app/docs/migration) to update safely.

> Version 1 becomes deprecated and unsupported since v2 is released.
New features and bug fixes will be pushed only to v2.

Please feel free to provide feedback or report any issues you encounter with the new version.
Thank you!

**Comment by ybrusentsov** [comment]
Special notification for active library users (contributors, issue openers, and open source repo users)
@ygrenzinger @sadiki-o @riyadelberkawy @onlyreddy @lucasluca @Baoing @rahulkeerthi @awell-health @Lilianada @mona-chen @sergioare

**Comment by yungshinobi** [comment]
Congratulations on the new release! (❁´◡`❁)
